src/sql/workbench/services/connection/browser/connectionDialogWidget.ts

```diff
@@ -336,6 +344,9 @@ export class ConnectionDialogWidget extends Modal {
 
 	private connect(element?: IConnectionProfile): void {
 		if (this._connectButton.enabled) {
+			this._telemetryService.createActionEvent(TelemetryKeys.TelemetryView.ConnectionDialog, TelemetryKeys.TelemetryAction.ConnectToServer).withAdditionalProperties(
```

Comment:
You're ok with this being fired even if the connection doesn't succeed?

Reply:
yes, whether the connection is succeed or not is not something we care about for this scenario. just want to understand the usage of different sources.